numcodecs/zarr3.py

```diff
@@ -362,19 +286,39 @@ def evolve_from_array_spec(self, array_spec: ArraySpec) -> AsType:
 
 
 # bytes-to-bytes checksum codecs
-CRC32 = _add_docstring(_make_checksum_codec("crc32", "CRC32"), "numcodecs.checksum32.CRC32")
```

Comment:
These codecs break the pattern of the ones above because they have `"numcodecs.checksum32.<name>"` instead of `"numcodecs.<lowercasename>.<name>"`. I ignored that, but it doesn't seem to matter whatsoever?? Suspicious.